doc/source/user_guide/tutorials/animate/animate_data.rst

```diff
@@ -0,0 +1,503 @@
+.. _ref_tutorials_animate_data:
+
+============
+Animate data
+============
+
+:bdg-warning:`MAPDL` :bdg-success:`LSDYNA`
+
+.. |Field| replace:: :class:`Field<ansys.dpf.core.field.Field>`
+.. |FieldsContainer| replace:: :class:`FieldsContainer<ansys.dpf.core.fields_container.FieldsContainer>`
+.. |Fields| replace:: :class:`Fields<ansys.dpf.core.field.Field>`
```

Comment:
```suggestion
```
Can we use `|Field|s` instead? Or `each |Field|`, or another variant

Reply:
okok